sos/plugins/selinux.py

```diff
@@ -27,7 +27,7 @@ def setup(self):
         ])
         self.add_cmd_output('sestatus')
 
-        state = self.get_command_output('getenforce')['output']
+        state = self.get_cmd_output_now('getenforce')['output']
```

Comment:
We will newly collect/store output of this command - a duplicate info resent in other cmd output(s), but it simplifies to find the info. Not sure if worth having `save=False` or not.